ten_quick_tips_live_coding.md

```diff
@@ -130,8 +130,10 @@ so resize the window(s) you use on your computer (drawing up the bottom) to comp
 Pay attention to the lighting:
 no lights should directly shine on the presenter's screen.
 
-It is even more important than when presenting slides that all learners can see the relevant portions of the screen.
-[FIXME: why?]
+It is even more important than when presenting slides
+that all learners can see the relevant portions of the screen
+as they may want to copy exactly what you have typed
+or see what you are pointing to.
```

Comment:
Probably worth adding a note that once you enlarge your font to be readable on a projected screen you'll have fewer columns and rows than you're used to, and you should design examples with this in mind (or at least test them). This is also a problem for IDEs: all the frames and panes take up space, but can't be shrunk.

Reply:
+1